src/server.cpp

```diff
@@ -354,42 +356,46 @@ void Server::shutdownModules() {
 
 static int statusToExitCode(const Status& status) {
     if (status.ok()) {
-#ifdef __linux__
-        return EX_OK;
-#elif _WIN32
-        return 0;
-#endif
+        return OVMS_EX_OK;
     } else if (status == StatusCode::OPTIONS_USAGE_ERROR) {
-#ifdef __linux__
-        return EX_USAGE;
-#elif _WIN32
-        return 3;
-#endif
+        return OVMS_EX_USAGE;
     }
-    return EXIT_FAILURE;
+    return OVMS_EX_FAILURE;
 }
 
 // OVMS Start
 int Server::start(int argc, char** argv) {
+#ifdef __linux__
```

Comment:
Remove ifdef __linux__ here.

Reply:
following comment the reduce code has already remove